modtools/create-unit.lua

```diff
@@ -25,7 +25,6 @@ local usage = [====[
 modtools/create-unit
 ====================
 Creates a unit.  Usage::
-
```

Comment:
I can see individual commits at https://github.com/DFHack/scripts/pull/14/commits, but that's it.